libc/math/common.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libc-tests
+ *
+ * Math tests common part.
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Adam Debek
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _TEST_MATH_COMMON_H
+#define _TEST_MATH_COMMON_H
+
+#define ITERR_FACTOR 100
+
+#define test_check_digLost(_fun, _x, _digLost, _acceptLoss) \
+	do { \
+		char errStr[150]; \
+		if (digLost > acceptLoss) { \
+			sprintf(errStr, _fun "(%g) gave lost of %.f significant digits of base 2 where " \
+								 "maximal acceptable loss is %.f signicant digits of base 2", \
+				_x, _digLost, _acceptLoss); \
+			TEST_FAIL_MESSAGE(errStr); \
+		} \
+	} while (0)
+
+#define test_check_digLost2(_fun, _x, _y, _digLost, _acceptLoss) \
+	do { \
+		char errStr[150]; \
+		if (digLost > acceptLoss) { \
+			sprintf(errStr, _fun "(%g, %g) gave lost of %.f significant digits of base 2 where " \
+								 "maximal acceptable loss is %.f signicant digits of base 2", \
+				_x, _y, _digLost, _acceptLoss); \
+			TEST_FAIL_MESSAGE(errStr); \
+		} \
+	} while (0)
+
+extern struct {
+	int maxPowTwoPi;
+	int maxPowTwo;
+	int dblSignif;
+} test_common;
```

Comment:
So the struct definition is in one place only
```suggestion
struct test_common_s {
	int maxPowTwoPi;
	int maxPowTwo;
	int dblSignif;
};

extern struct test_common_s test_common;
```